suite-native/module-onboarding/src/screens/WelcomeScreen.tsx

```diff
@@ -25,49 +29,112 @@ const imageContainerStyle = prepareNativeStyle(() => ({
     top: 0,
 }));
 
+const contentStyle = prepareNativeStyle(utils => ({
+    width: '100%',
+    backgroundColor: utils.colors.backgroundAlertRedBold,
+    alignItems: 'center',
+}));
+
+const cardStyle = prepareNativeStyle(() => ({
+    marginHorizontal: 16,
+    marginTop: 16,
+    padding: 24,
+    borderRadius: 20,
+    flex: 1,
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    borderWidth: 1,
+}));
+
 export const WelcomeScreen = () => {
+    const { translate } = useTranslate();
+
+    const { isUsbDeviceConnectFeatureEnabled: isUsbConnectEnabled } =
+        useIsUsbDeviceConnectFeatureEnabled();
+
     const navigation =
         useNavigation<
             StackNavigationProps<OnboardingStackParamList, OnboardingStackRoutes.Welcome>
         >();
+
     const { applyStyle } = useNativeStyles();
+
     const colorScheme = useActiveColorScheme();
 
     const handleRedirect = () => {
-        navigation.navigate(OnboardingStackRoutes.TrackBalances);
+        navigation.navigate(
+            isUsbConnectEnabled
+                ? OnboardingStackRoutes.ConnectTrezor
+                : OnboardingStackRoutes.TrackBalances,
+        );
     };
 
+    const isDarkMode = colorScheme === 'dark';
+
     const getImageSource = () => {
-        if (colorScheme === 'dark') {
+        if (isDarkMode) {
             return require('../assets/darkRectangles.png');
         }
         return require('../assets/rectangles.png');
     };
 
+    const borderColor = isDarkMode ? '#00000099' : '#FFFFFF99';
```

Comment:
We shouldn't just add colors like this. If there is requirement to add more colors, we should add them through theming. Can't we use some of the existing colors for this?

Reply:
Done